templates/image-builder.yml

```diff
@@ -215,3 +220,9 @@ parameters:
   - name: DISTRIBUTIONS_DIR
     description: Directory which contains json files detailing available distributions, their repositories, and their packages
     value: "/app/distributions"
+  - name: OSBUILD_GCP_REGION
+    description: Region in GCP to upload to
+    value: "us-east4"
+  - name: OSBUILD_GCP_BUCKET
+    description: Bucket in GCP to upload to
+    value: "image-upload-bkt-us-east4"
```

Comment:
This should be just "image-upload-bkt-us". The Storage bucket is multi-regional, so there is no need to have one per each region. There will be just one bucket for each - US, EU, ASIA (https://cloud.google.com/about/locations#multi-region)